posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,78 @@
+<!--
+.. title: Files Improvements Spyder4
+.. slug: Files-Improvements
+.. date: 2019-11-01 12:00:00 UTC-05:00
+.. author: Juanita Gomez & Gonzalo Peña
+.. tags: Labs, Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+# Files Improvements Spyder4
+
```

Comment:
Perhaps a sentence more for introduction would be good. Something like: "Spyder 4.0 is almost ready. It has been in the making for 1.5 (?) years, and contains lots of interesting new features. In this post we will focus on ...."